lib/assert/test.rb

```diff
@@ -137,13 +139,17 @@ def capture_io
       StringIO.new(@output, "a+")
     end
 
-    def name_from_context(name)
-      [ self.context_class.description,
-        name
-      ].compact.reject{|p| p.empty?}.join(" ")
+    def name_file_line_from_context(context_info, name)
+      [ [ context_info.klass.description,
+          name
+        ].compact.reject{|p| p.empty?}.join(" "),
```

Comment:
@kellyredding - I think you could do `.reject(&:empty?)` here. Not 100% sure. Not a big deal either way.